### PR TITLE
Add ETL script for schedulers

### PR DESCRIPTION
**Pull request by klahnakoski**

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -33,10 +33,15 @@ If you belong to the [CI-A Github team](https://github.com/orgs/mozilla/teams/ci
...
-### Retrieving secrets locally
+## Run task on local machine 
+
+
+    set TASKCLUSTER_ROOT_URL=https://community-tc.services.mozilla.com
+
+### Retrieving secrets locally (linux only)
 
 To test *locally* that your script can fetch secrets you will have to [download a binary](https://github.com/taskcluster/taskcluster/tree/master/clients/client-shell#readme)
-to set up your credentials. Unfortunately, this only works for Linux and Windows (filed [issue](https://github.com/armenzg/smart-scheduling/issues/1)).
```
Comment (armenzg) [inline scripts/README.md:39]: I meant to say "Linux and Mac OS X"

File: scripts/README.md [hunk 2]
```diff
@@ -33,10 +33,15 @@ If you belong to the [CI-A Github team](https://github.com/orgs/mozilla/teams/ci
 Open [this secret](https://community-tc.services.mozilla.com/secrets/project%2Fcia%2Fgarbage%2Ffoo) and try to see it. If you can see the
 secret you can then create your own.
 
-### Retrieving secrets locally
+## Run task on local machine 
+
+
+    set TASKCLUSTER_ROOT_URL=https://community-tc.services.mozilla.com
```
Comment (armenzg) [inline scripts/README.md]: Why do you need to set this?

**Review by armenzg — COMMENTED** [review]
Just some questions for you.

FYI you can set a gitignore in your home directory that applies to any checkouts. I use that for few things.

Are you sure you want to land all your vendor libraries instead of using Python packages? It will get on the way of `git grep` or IDEs indexing it.

If you still want them please land them under a vendor directory at the top of the project so all future scripts can make reference to it.

File: scripts/README.md (hunk 1 above)
Comment (klahnakoski) [reply scripts/README.md:39]: fixed

File: scripts/README.md (hunk 2 above)
Comment (klahnakoski) [reply scripts/README.md]: this is required for the taskcluster library to know where to go

**Comment by klahnakoski** [comment]
> FYI you can set a gitignore in your home directory that applies to any checkouts. I use that for few things.

You can add  anything you want to the `.gitignore`:  I am only replacing it because it is full of ignored files we probably do not have.

> Are you sure you want to land all your vendor libraries instead of using Python packages? 

Yes, this is more flexible than going through library update cycle (which includes updating the library code, fixing tests, releasing to pip, waiting for pip, getting hashes)

> If you still want them please land them under a vendor directory at the top of the project so all future scripts can make reference to it.

I do not think this should happen:  These vendor directories are for this particular script they need not change once deployed.

**Pull request merged**